week_3/rds.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_db_instance" "week3db" {
+  allocated_storage      = 10
+  engine                 = "postgres"
+  engine_version         = "13.5"
+  name                   = "week3db"
+  username               = "week3user"
+  password               = "hello12345"
```

Comment:
Instead of hardcoding password, it can be created using a terraform resource: https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/password